backend/presentation/tests.py

```diff
@@ -0,0 +1,325 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from presentation.models import Presentation, Tag
+from presentation.serializers import PresentationSerializer, TagSerializer
+
+
+User = get_user_model()
+
+
+class TestPresentationViewSet(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.admin = User.objects.create_user(
```

Comment:
You could consider using `factory-boy` package used for creating model factories.